### DISCUSSION TITLE
WoW Support: Middle-Click and move, Left- & Right-Click

### DISCUSSION THREAD

**Discussion by UntoldTalesNWN** [rxhanson/Multitouch-Community#199] [Q&A] [answered]
I am currently assessing if I can use my Magic Mouse for World of Warcraft gaming. I got the Middle-Click working, but when I do so, I cannot move the screen (which it usually should do). Furthermore, I couldn't find a way to emulate a simulatneous Left- and Right-Click. Any help is highly appreciated. Thanks in advance!

**Comment by rxhanson** [comment, marked as answer]
Thanks for reporting. 

The default middle click behavior is geared to work best with 3D modeling applications. To get the behavior that works with games like WoW, you can execute the following terminal command:

defaults write com.brassmonkery.Multitouch mutateMouseDragEvents -bool true

Then restart the app.

I'll add the simultaneous left & right click to my backlog, but I don't have a timeframe for when I would get to it.

**Reply by UntoldTalesNWN** [reply]
Wow! That's a game changer. Now you actually can play WoW (and many other MMO with mouse movement) perfectly fine with a Magic Mouse. I guess it may be worth thinking about adding such "easy" gaming options somewhere more prominent. Thanks a lot!
